src/app/portlets/dot-edit-page/content/components/dot-edit-content-view-as-toolbar/dot-edit-content-view-as-toolbar.component.spec.ts

```diff
@@ -235,6 +236,7 @@ describe('DotEditContentViewAsToolbarComponent', () => {
                     JSON.parse(JSON.stringify(mockDotRenderedPage)),
                     PageMode.PREVIEW
                 );
+                spyOn(dotLicenseService, 'isEnterprise').and.returnValue(of(true));
                 fixtureHost.detectChanges();
 
                 whatsChanged = de.query(By.css('p-checkbox'));
```

Comment:
I think we should have only one test here that check what component should be not null in case of EE license, right now we have tree test for it, also the new test should check the 'what change' checkbox